content/applications/hr/payroll.rst

```diff
@@ -108,6 +108,17 @@ The contract template forms the basis of an offer, and can be modified for speci
 employees, when necessary. If a contract template is created or modified in the *Payroll*
 application, the changes are also reflected in the *Recruitment* application.
 
+.. important::
+   To access contract templates, the :guilabel:`Salary Configurator` module **must** be installed.
+
+   To install this module, navigate to the *Apps* application. In the search bar, delete the
+   :guilabel:`Apps` filter, then type in :guilabel:`Salary Configurator`.
+
+   Click :guilabel:`Install` on the :guilabel:`Salary Configurator` module, and that module, along
+   with the :guilabel:`Salary Configurator - Holidays` and :guilabel:`Salary Configurator - Payroll`
+   modules install as well. Once the modules are installed, the database reverts to the main
+   dashboard.
+
 To view all the current contract templates in the database, navigate to :menuselection:`Payroll app
 --> Configuration --> Contracts: Templates`.
```

Comment:
with the required modules installed, I'm not seeing this menu option available...

![Screenshot 2024-05-13 151740](https://github.com/odoo/documentation/assets/73958186/0187e306-f4f7-417e-a12d-2297765af6ca)

Might be worth double-checking the accuracy of this during your revisions...

Reply:
I tested this in a blank database with only the Employees app and Payroll apps installed, and the modules specified here- and that made it show up:
![image](https://github.com/odoo/documentation/assets/121518652/9e54229c-d76a-4598-a33e-4a2b0e51d82c)

I am not sure why you aren't seeing it in here- so I assume itmust be a runbot issue since I can make it appear on a clean database.  

I also checked a runbot and the module was already installed and I saw it - perhaps they changed permissions on the runbot you were on?